src/hotspot/share/runtime/frame.inline.hpp

```diff
@@ -104,6 +104,21 @@ inline CodeBlob* frame::get_cb() const {
   return _cb;
 }
 
+inline const ImmutableOopMap* frame::get_oop_map() const {
+  if (_cb == nullptr) return nullptr;
+  if (_cb->oop_maps() != nullptr) {
```

Comment:
Could be shorter: `if (_cb == nullptr || _cb->oop_maps() == nullptr) return nullptr;`

Reply:
Yes, that's better.